packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -159,7 +162,7 @@ export default function (
             sourceMap: !!(task.sourceMappings || task.sourceMap),
           });
 
-          _.extend(task.options, taskOpts);
+          extend(task.options, taskOpts);
```

Comment:
`Object.assign`, or is it intended that `taskOpts` prototype options are copied? How do you even get things in `taskOpts`' prototype since it comes from JSON?